qa/L0_sagemaker/sagemaker_multi_model_test.py

```diff
@@ -280,6 +284,27 @@ def test_sm_5_model_unload(self):
             r.status_code, 404,
             "Expected status code 404, received {}".format(r.status_code))
 
+    def test_sm_6_ensemble_model(self):
+        # Load ensemble model
+        request_body = {"model_name": self.model3_name, "url": self.model3_url}
+        headers = {"Content-Type": "application/json", "X-Amzn-SageMaker-Target-Model": f"{self.model3_name}"}
+        r = requests.post(self.url_mme_,
+                          data=json.dumps(request_body),
+                          headers=headers)
+        time.sleep(10)  # wait for model to load
+        self.assertEqual(
+            r.status_code, 200,
+            "Expected status code 200, received {}".format(r.status_code))
```

Comment:
Should also send an inference to make sure the ensemble is properly loaded?

Reply:
Updated the test to include the inference request as well, thx